spec/shameless/model_spec.rb

```diff
@@ -171,6 +171,28 @@
     end
   end
 
+  describe '#where' do
```

Comment:
Could we add something similar in `index_spec.rb` that queries an explicit index?